src/status_im/contexts/chat/messenger/messages/view.cljs

```diff
@@ -1,19 +1,32 @@
 (ns status-im.contexts.chat.messenger.messages.view
   (:require
+    [clojure.string :as string]
     [quo.theme :as quo.theme]
     [react-native.core :as rn]
     [react-native.platform :as platform]
     [react-native.reanimated :as reanimated]
     [react-native.safe-area :as safe-area]
     [reagent.core :as reagent]
-    [status-im.contexts.chat.messenger.composer.view :as composer.view]
+    [status-im.contexts.chat.messenger.composer.view :as composer]
+    [status-im.contexts.chat.messenger.messages.contact-requests.bottom-drawer.view :as
+     contact-requests.bottom-drawer]
     [status-im.contexts.chat.messenger.messages.list.style :as style]
     [status-im.contexts.chat.messenger.messages.list.view :as list.view]
     [status-im.contexts.chat.messenger.messages.navigation.view :as messages.navigation]
+    [status-im.contexts.chat.messenger.messages.scroll-to-bottom.view :as scroll-to-bottom]
     [status-im.contexts.chat.messenger.placeholder.view :as placeholder.view]
     [status-im.feature-flags :as ff]
     [utils.re-frame :as rf]))
 
+(defn- footer
+  []
+  (let [current-chat-id       (rf/sub [:chats/current-chat-id])
+        able-to-send-message? (rf/sub [:chats/able-to-send-message?])]
+    (when-not (string/blank? current-chat-id)
+      (if able-to-send-message?
+        [composer/view]
+        [contact-requests.bottom-drawer/view {:contact-id current-chat-id}]))))
```

Comment:
I think we should have a subscription to check if the `:current-chat-id` exists. Checking it by using `string/blank` is hard to remember for other non involved devs.